### PR TITLE
Add note

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/10214

source https://learn.microsoft.com/en-us/officeupdates/current-channel-preview#outlook-88

### CODE REVIEW

File: Teams/set-up-phone-system-voicemail.md [hunk 1]
```diff
@@ -140,6 +140,10 @@ If you’re an administrator, you can use the following diagnostic tool to valid
 2. In the Run diagnostic pane, enter the email of the user who is having issues receiving voicemail in the **Username or Email** field, and then select **Run Tests**.
 3. The tests return the best next steps to address any tenant or policy configurations to validate that the user is properly configured to receive voicemail in Teams.
 
+>[!NOTE]
+> There could be certain situations where the voicemail form does not get displayed in the UI in Outlook Desktop, the following registry key needs to be set: [HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\Outlook\Addins] "AllowVoicemailForm"=dword:00000001, for users, enterprises, and organizations that want the Voicemail form to appear.
+
```
Comment (JohanFreelancer9) [inline Teams/set-up-phone-system-voicemail.md:145]:
```suggestion
> [!NOTE]
> There could be certain situations where the voicemail form is not displayed in the UI in Outlook Desktop. The following registry key needs to be set for users, enterprises, and organizations that want the Voicemail form to appear: [HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\Outlook\Addins] "AllowVoicemailForm"=dword:00000001.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mkbond007** [comment]
@JohanFreelancer9 I'm talking with PMs about this update. I will make edits and/or give an update once I get a response.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@mkbond007 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by mkbond007** [comment]
@scanum @get-itips @JohanFreelancer9 Still talking internally about whether or not to add this/what to add if so.

**Comment by mkbond007** [comment]
This note doesn't meet the criteria for adding to this article, however, it's up to you if it's added to troubleshooting content.

**Comment by get-itips** [comment]
> This note doesn't meet the criteria for adding to this article, however, it's up to you if it's added to troubleshooting content.

No problem at all :) I'll close it then.

**Pull request closed without merging**